dis_snek/client/client.py

```diff
@@ -606,21 +603,23 @@ def start(self, token) -> None:
 
         """
         try:
-            self.loop.run_until_complete(self.login(token))
-            self.loop.run_until_complete(self._connection_state.start())
+            asyncio.run(self.astart(token))
         except KeyboardInterrupt:
-            self.loop.run_until_complete(self.stop())
+            # ignore, cus this is useless and can be misleading to the
+            # user
```

Comment:
just a nitpick, not sure why a newline is needed for the comment here

Reply:
![line](https://user-images.githubusercontent.com/82430093/156933028-2b1f3763-5e74-48e1-9542-220a50d00eb8.png)
![pep8](https://user-images.githubusercontent.com/82430093/156933051-dee786f0-9f2a-42d3-b188-d1efb66540fb.png)

I'm a pep-8 boi

Reply:
https://github.com/Discord-Snake-Pit/Dis-Snek/blob/master/pyproject.toml#L22

Reply:
Loll then like 70% of our comments are invalid format ripp